src/main/java/org/broadinstitute/consent/http/service/SupportRequestService.java

```diff
@@ -105,24 +68,25 @@ public void postTicketToSupport(SupportTicket ticket, AuthUser authUser) throws
         }
     }
 
-
     /**
-     * Creates and sends a support ticket for a user requesting an unfamiliar institution and/or signing official, if provided
+     * Creates and sends a support ticket for a user selecting an existing or requesting an unfamiliar institution
+     * and/or signing official, if provided
      *
      * @param userUpdateFields A UserUpdateFields object containing update information for the user
      * @param user             The user requesting the institution and/or signing official
-     * @param authUser         AuthUser object used to build POST request
      */
-    public void handleSuggestedUserFieldsSupportRequest(UserUpdateFields userUpdateFields, User user, AuthUser authUser) {
+    public void handleInstitutionSOSupportRequest(UserUpdateFields userUpdateFields, User user) {
         if (Objects.nonNull(userUpdateFields) && Objects.nonNull(user)) {
-            String suggestedInstitution = userUpdateFields.getSuggestedInstitution();
-            String suggestedSigningOfficial = userUpdateFields.getSuggestedSigningOfficial();
+            boolean updateFieldProvided = Objects.nonNull(userUpdateFields.getSuggestedInstitution())
+                    || Objects.nonNull(userUpdateFields.getInstitutionId())
+                    || Objects.nonNull(userUpdateFields.getSuggestedSigningOfficial())
+                    || Objects.nonNull(userUpdateFields.getSelectedSigningOfficialId());
 
-            // Only create and send ticket if either a suggestedInstitution or suggestedSigningOfficial has been provided
-            if (Objects.nonNull(suggestedInstitution) || Objects.nonNull(suggestedSigningOfficial)) {
+            //only send ticket if an institution or signing official is provided; ignore otherwise
+            if (updateFieldProvided) {
                 try {
-                    SupportTicket ticket = createSuggestedUserFieldsTicket(userUpdateFields, user);
-                    postTicketToSupport(ticket, authUser);
+                    SupportTicket ticket = supportTicketCreator.createInstitutionSOSupportTicket(userUpdateFields, user, configuration.postSupportRequestUrl());
```

Comment:
The url is included in the comment body (based on the template from [DUOS-1890](https://broadworkbench.atlassian.net/browse/DUOS-1890)). Is there a preference between passing in the url here vs having a configuration field in `SupportTicketCreator`?

Reply:
If we're moving construction details to the creator class, then it does make sense to add a config there as well.